### DISCUSSION TITLE
Hybridising mx.mod.Module?

### DISCUSSION THREAD

**Discussion by ghost** [apache/incubator-mxnet#15075] [Q&A]
How can we hybridize the mx.mod.Module? Is it possible?

**Comment by frankfliu** [comment]
@mxnet-label-bot add [question]

**Comment by leleamol** [comment]
@vizakshat   The mx.mod.Module is part of Module API. The Module API provides a wrapper on Symbol API.  Here is more information about Module https://mxnet.incubator.apache.org/versions/master/tutorials/basic/module.html

The Gluon is a separate paradigm that allows user to construct neural networks with higher flexibility. The hybridize method optimizes the execution of training workflow. This document explains the hybridization is great detail https://www.d2l.ai/chapter_computational-performance/hybridize.html?highlight=hybridize

Going by these documents, it is not possible to invoke hybridize in mx.mod.Module.  I hope this answers your question. If you have further questions, I would recommend posting them on MXNet forum at https://discuss.mxnet.io/

@mxnet-label-bot add [Gluon, Module, Pending Requester Info]
